backend/app/utils.py

```diff
@@ -394,6 +397,33 @@ def appstream2dict(appstream_url=None) -> tuple[dict[str, dict], dict[str, dict]
     return apps, apps_locale
 
 
+def get_content_rating_details(content_rating: dict, locale: str) -> dict:
```

Comment:
Did you work this into the update routine at some point already?

Reply:
No, because I wasn't sure how to. See my comment above:

> libappstream also provides localized descriptions of all content ratings. I was not sure how to integrate this with flathub's approach to i18n. We could either try to hook into that. Or we could duplicate the code.